tests/test_workflow_api.py

```diff
@@ -969,7 +969,7 @@ async def test_delete_workflow_failure(
     """
     GIVEN a Workflow in a relationship with a Job
     WHEN we try to DELETE that Workflow
-    THEN we fail with a 422
+    THEN we succeed
```

Comment:
This test is not working correctly: it tests the deletion of a workflow which is not related to any job)

Reply:
Also, to be added: after deletion, we should assert that job has `workflow_id` equal to `None`. And we should also assert something on the `job.workflow` attribute.